src/main/java/org/reaktivity/nukleus/oauth/internal/OAuthRealms.java

```diff
@@ -108,6 +129,13 @@ public JsonWebKey supplyKey(
         return keysByKid.get(kid);
     }
 
+    private OAuthRealm newOAuthRealm(
+        String realmName)
+    {
+        assert nextRealmBit < MAX_REALMS;
+        return new OAuthRealm(realmName, nextRealmBit++);
+    }
```

Comment:
Move `nextRealmBit++` to constructor implementation, and remove second parameter from constructor.
Move `assert` to constructor implementation as well.
Then, this method is no longer needed, reference `OAuthRealm::new` instead of `this::newOAuthRealm`.